docs-new/docs/graphql/core/databases/postgres/schema/table-relationships/index.mdx

```diff
@@ -0,0 +1,66 @@
+---
+description: Manage relationships between Postgres tables/views in Hasura
+keywords:
+  - hasura
+  - docs
+  - postgres
+  - schema
+  - relationship
+slug: index
+---
+
+# Postgres: Relationships between tables/views
+
+## Introduction
+
+To make [nested object queries](/graphql/core/databases/postgres/queries/nested-object-queries.mdx), the
+tables/views in your database need to be connected via relationships.
+
+Let's say we have the following tables in our database: `authors`,
+`passport_infos`, `articles` and `tags`.
+
+## Table relationships
+
+The tables/views in any relational database are typically related to
+each other via one of the following types of table relationships:
+
+|Type           | Example                        | Meaning                                                                              |
+|---------------|--------------------------------|--------------------------------------------------------------------------------------|
+|`one-to-one`   | `authors` and `passport_infos` | an `author` can have one `passport_info` \| a `passport_info` can have one `owner`   |
+|`one-to-many`  | `authors` and `articles`       | an `author` can have many `articles` \| an `article` can have one `author`           |	
+|`many-to-many` | `articles` and `tags`          | an `article` can have many `tags` \| a `tag` can have many `articles`                |
```

Comment:
The `Meaning` column of this table seems to have list items in UI.

Please try converting to HTML if this syntax doesn't work out for adding list items.

P.S. Don worry about indentation as it might not parse correctly. Please leave an empty line before an after if using markdown syntax.

<img width="983" alt="Screenshot 2022-02-02 at 7 15 15 PM" src="https://user-images.githubusercontent.com/22497932/152165708-e1bb29f1-2f80-4dbe-9d52-543c40078a99.png">

Reply:
Done